#standard metadata for higher-level flow data types
uint32 seq
time stamp
string frame_id
uint32 id
bool is_rtr
bool is_extended
bool is_error
uint8 dlc
uint8[8] data
